src/analysis/retail/evoker/preservation/modules/talents/DreamFlight.tsx

```diff
@@ -60,6 +90,93 @@ class DreamFlight extends Analyzer {
     };
   }
 
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <b>
+          <SpellLink id={TALENTS_EVOKER.DREAM_FLIGHT_TALENT.id} />
+        </b>{' '}
+        is a powerful healing CD that does an initial heal and leaves a powerful HoT on all targets
+        hit. You should try to use it to cover high damage periods where your raid is clumped up in
+        order to maximize its effectiveness. If used when your allies are spread out, it will do
+        very little healing to the point where spread out fights make this talent unusable.
+      </p>
+    );
+    const data = (
+      <div>
+        <RoundedPanel>
+          <strong>
+            <SpellLink id={TALENTS_EVOKER.DREAM_FLIGHT_TALENT} /> cast efficiency
+          </strong>
+          <div className="flex-main chart" style={{ padding: 15 }}>
+            {this.subStatistic()}
+          </div>
+          <br />
+          {this.castInfo.map((info, idx) => {
+            const header = (
+              <>
+                <SpellLink spell={TALENTS_EVOKER.DREAM_FLIGHT_TALENT} /> @{' '}
+                {this.owner.formatTimestamp(info.castTime)}
+              </>
+            );
+            const checklistItems: CooldownExpandableItem[] = [];
+            let targetsHitPerf = QualitativePerformance.Good;
+            let overhealingPerf = QualitativePerformance.Good;
+            const percentHit = info.totalHit / this.combatants.playerCount;
+            if (percentHit < 0.65) {
+              targetsHitPerf = QualitativePerformance.Fail;
+            } else if (percentHit < 0.75) {
+              targetsHitPerf = QualitativePerformance.Ok;
+            }
+            checklistItems.push({
+              label: (
+                <>
+                  % of raid hit with <SpellLink spell={TALENTS_EVOKER.DREAM_FLIGHT_TALENT} />
+                </>
+              ),
+              result: <PerformanceMark perf={targetsHitPerf} />,
+              details: <>{formatPercentage(percentHit)}%</>,
+            });
+            const overhealPercent = info.overhealing / (info.overhealing + info.healing);
+            if (overhealPercent > 0.6) {
+              overhealingPerf = QualitativePerformance.Fail;
+            } else if (overhealPercent > 0.5) {
```

Comment:
50% overheal seems high for OK on a raid CD, esp. after the health change but I'll defer to your judgment